datafusion/core/tests/sqllogictests/test_files/arrow_typeof.slt

```diff
@@ -303,3 +307,25 @@ select arrow_cast(interval '30 minutes', 'Duration(Second)');
 
 query error DataFusion error: Error during planning: Cannot automatically convert Utf8 to Duration\(Second\)
 select arrow_cast('30 minutes', 'Duration(Second)');
+
+
+## Timestamptz
+
+query P
+select arrow_cast(timestamp '2000-01-01T00:00:00', 'Timestamp(Nanosecond, Some( "+00:00" ))');
+----
+2000-01-01T00:00:00Z
+
+query P
+select arrow_cast(timestamp '2000-01-01T00:00:00', 'Timestamp(Nanosecond, Some( "+08:00" ))');
+----
+2000-01-01T08:00:00+08:00
+
+# once https://github.com/apache/arrow-rs/issues/1936 fixed, please update this query
+query P
+select arrow_cast(timestamp '2000-01-01T00:00:00Z', 'Timestamp(Nanosecond, Some( "+08:00" ))');
+----
+2000-01-01T08:00:00+08:00
```

Comment:
@tustvold thank you
added this case to notify users once the kernel casting issue fixed